src/libraries/System.Runtime.Extensions/tests/System/Numerics/BitOperationsTests.cs

```diff
@@ -887,5 +887,49 @@ public static void BitOps_RoundUpToPow2_nuint_64(ulong value, ulong expected)
         {
             Assert.Equal(expected, BitOperations.RoundUpToPowerOf2((nuint) value));
         }
+
+        [Theory]
+        [InlineData(0, 0, 0)]
+        [InlineData(0, 120, 4215344322)]
+        [InlineData(0, byte.MaxValue, 2910671697)]
+        [InlineData(123, byte.MaxValue, 1164749927)]
+        public static void BitOps_Crc32C_byte(uint crc, byte data, uint expected)
+        {
+            var obtained = BitOperations.Crc32C(crc, data);
```

Comment:
Use the concrete type instead of `var`.

Same below.

Reply:
done